src/model/configuration_example/training/training_v1.yaml

```diff
@@ -11,4 +11,11 @@ batch_size: 2
 grad: 4
 max_tokens: 2048
 num_shards: 1024
-num_workers: 2
\ No newline at end of file
+num_workers: 2
+epochs: 1
+start_epochs: 0
```

Comment:
What the meaning of start steps and start epoch. Can you explain in the yaml comments?